.github/workflows/ci.yaml

```diff
@@ -114,6 +114,10 @@ jobs:
           key: ${{ runner.os }}-go-${{ hashFiles('go.sum') }}
           restore-keys: |
             ${{ runner.os }}-go-
+      - name: Make Sure That OpenAPI code has been generated
```

Comment:
The name is not fully accurate. It should be something like `Make sure there is no uncommited autogenerated code` or something like that

Reply:
I like the way you worded it.

Reply:
Yes. That's true. I've changed the script name as well.